src/transactions/SetOptionsOpFrame.cpp

```diff
@@ -164,6 +166,17 @@ SetOptionsOpFrame::doApply(AbstractLedgerTxn& ltx)
         account.flags = account.flags | *mSetOptions.setFlags;
     }
 
+    // check clawback state after account.flags has changed
+    if (mSetOptions.setFlags || mSetOptions.clearFlags)
+    {
+        if ((account.flags & AUTH_CLAWBACK_ENABLED_FLAG) &&
+            (account.flags & AUTH_REVOCABLE_FLAG) == 0)
```

Comment:
extra parenthesis for clarity
```
        if ((account.flags & AUTH_CLAWBACK_ENABLED_FLAG) &&
            ((account.flags & AUTH_REVOCABLE_FLAG) == 0))
```
or maybe just
```
        if ((account.flags & AUTH_CLAWBACK_ENABLED_FLAG) &&
            ~(account.flags & AUTH_REVOCABLE_FLAG))
```

Reply:
Done